stratum/glue/net_util/bits_test.cc

```diff
@@ -226,9 +226,9 @@ TEST_F(BitsTest, BitCountingRandom) {
     float p = RandomFloat();
     int nbits = 0;
     uint32 n = 0;
-    for (int i = 0; i < 32; i++) {
+    for (int j = 0; j < 32; j++) {
```

Comment:
wow...using a variable `i` in an inner for loop that also uses `i` -- good catch from compiler